lib/ace/socket.ex

```diff
@@ -85,4 +85,18 @@ defmodule Ace.Socket do
   def send({:ssl, socket}, message) do
     :ssl.send(socket, message)
   end
+  
+  # defp client_info(soc) do
+  #   "soc:\n#{inspect(soc)}\n"
+  # end
+
+  def client_info({:ssl, socket}) do
```

Comment:
This function doesn't handle failures. It also only returns a string rather than any accessible information.

I am going to close for the moment

Reply:
The option would be to extend `Raxx.Response` and/or `Raxx.Request` with the accessible information.
I don't know if that is the best way to do this.

Reply:
I was wondering about putting this metadata in the process dictionary

Reply:
Sound good.
So what I think is , if it wont cost much in terms of processing overhead, and memory, then the process dictionary may be the right place for all the additional information that one may need for ACE + Raxx.

We may also add response timing info, directly into the process dictionary.
We may need to come up with format/naming which would be intuitive for accessing this info as well.